src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail_controller.js

```diff
@@ -108,6 +106,18 @@ export default class ReplicationControllerDetailController {
      * @export {boolean}
      */
     this.eventsOrder = DOWNWARDS;
+
+    /**
+     * If true then sidebar will be displayed, otherwise it won't.
+     * @export {boolean}
+     */
+    this.displaySidebar = this.mdMedia('gt-sm');
+
+    /**
+     * Binding resize listener, which switches displaySidebar boolean variable.
+     */
+    angular.element(this.window)
```

Comment:
One step further might be to put ctrl.mdMedia("gt-sm") into a method in controller like 
hasMinimumSize() {
    return this.mdMedia('gt-sm');
}
Because the logic is doubled in the html code.